### PR TITLE
[NayNay] Create Account Refactor + Testing

### LINKED ISSUE #148: createAccount
```js
createAccount({ name, seed, path? }): Promise<{name, address, data: EntropyAccountData}>
```

### PR DESCRIPTION
**Pull request by rh0delta**
- converted creating an account to pure function
- added tests for creating an account
- renamed some files for organization and consistency

closes #148

### CODE REVIEW

**Comment by rh0delta** [comment]
NOTE: Build will consistently fail until https://github.com/entropyxyz/sdk/pull/395 is merged in the sdk

**Review by mixmix — DISMISSED** [review]
Great tidy-up.
Only changes are lines I think you can safely remove from tests!

**Comment by rh0delta** [comment]
@mixmix this pr is ready for re-review

File: CHANGELOG.md
Comment (frankiebee) [inline CHANGELOG.md:1]: 💛

**Review by frankiebee — APPROVED** [review]
Woop woop!

**Pull request merged**